bin/get_countries.rb

```diff
@@ -33,11 +33,12 @@
 results_hash = JSON.parse results_str
 source_array = results_hash.dig "results", "bindings"
 
-dest_array = source_array.inject([]) do |dest_array, result|
```

Comment:
Suddenly occurs to me that the removal of this line means that [Line 44] will explode with a `NameError`. Not that we were using this, but still 😈